Bottom: 4d00360b7a912e4a06b97e6a747e8fd1bbb85958
Top:    a6a845fe07b090bf78a6cf6746607a7779b2e3ef
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-09 15:42:43 -0500

Add new dependency, bc, to Dockerfile


---
diff --git a/docker/buildbot-worker-ptxdist/Dockerfile b/docker/buildbot-worker-ptxdist/Dockerfile
index ee6bfeb5..b14d49c4 100644
--- a/docker/buildbot-worker-ptxdist/Dockerfile
+++ b/docker/buildbot-worker-ptxdist/Dockerfile
@@ -22,6 +22,7 @@ RUN apt-get update \
 &&  DEBIAN_FRONTEND=noninteractive \
     apt-get -y install -q \
         autoconf \
+        bc \
         bison \
         build-essential \
         ccache \
